src/manager.cpp

```diff
@@ -381,6 +385,9 @@ static std::map<pid_t, std::string> start_modules(Config& config, MQTTAbstractio
 
         mqtt_abstraction.register_handler(topic, module_it->second.token, QOS::QOS2);
 
+        std::string error_topic = fmt::format("{}/+/error/#", module_name);
```

Comment:
Do we have to use wildcards here?

Reply:
I changed it to adding each topic one by one. A restructuring of the topic structure in the future could increase performance here probably